### PR TITLE
Post release 1.7.3

### PR DESCRIPTION
**Pull request by beta-ziliani**

### CODE REVIEW

File: _releases/2023-03-03-1.7.3-released.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+We are releasing the third patch release of the [1.7](/2023/01/09/1.7.0-released/) series.
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.7.3) and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for installation instructions.
+
+This patch includes a bugfix for a regression introduced in 1.7.0 and several bugfixes for supporting the PCRE2 regex library.
+
+The regression affected the compiler. Prior to 1.7.0 it wasn't checking that numeric arguments of generics were matching. The fix in 1.7.0 however [affected the inference](https://github.com/crystal-lang/crystal/issues/13110) of numeric arguments.
+
+The bugfixes for PCRE2 are important to help migrating projects into the newly supported library. Mind that support for the old regex library will gradually [fade out](https://crystal-lang.org/2023/03/02/crystal-is-upgrading-its-regex-engine/).
```
Comment (straight-shoota) [inline _releases/2023-03-03-1.7.3-released.md:18]: We should mention that this is only relevant if you explicitly opt-in for the preview:
```suggestion
The bugfixes for PCRE2 are important to help migrating projects into the newly supported library when testing it with `-Duse_pcre2`. Mind that support for the old regex library will gradually [fade out](https://crystal-lang.org/2023/03/02/crystal-is-upgrading-its-regex-engine/).
```

**Review by straight-shoota — APPROVED** [review]

File: _releases/2023-03-03-1.7.3-released.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+thumbnail: +
+author:
+---
+
+We are releasing the third patch release of the [1.7](/2023/01/09/1.7.0-released/) series.
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.7.3) and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for installation instructions.
+
+This patch includes a bugfix for a regression introduced in 1.7.0 and several bugfixes for supporting the PCRE2 regex library.
```
Comment (beta-ziliani) [inline _releases/2023-03-03-1.7.3-released.md:14]:
```suggestion
This patch includes a bugfix for a regression introduced in 1.7.0 and several bugfixes and a performance regression fix for supporting the PCRE2 regex library.
```

File: _releases/2023-03-03-1.7.3-released.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+The regression affected the compiler. Prior to 1.7.0 it wasn't checking that numeric arguments of generics were matching. The fix in 1.7.0 however [affected the inference](https://github.com/crystal-lang/crystal/issues/13110) of numeric arguments.
+
+The bugfixes for PCRE2 are important to help migrating projects into the newly supported library when testing it with `-Duse_pcre2`. Mind that support for the old regex library will gradually [fade out](https://crystal-lang.org/2023/03/02/crystal-is-upgrading-its-regex-engine/).
```
Comment (beta-ziliani) [inline _releases/2023-03-03-1.7.3-released.md:18]:
```suggestion
The fixes for PCRE2 are important to help migrating projects into the newly supported library when testing it with `-Duse_pcre2`. Mind that support for the old regex library will gradually [fade out](https://crystal-lang.org/2023/03/02/crystal-is-upgrading-its-regex-engine/).
```

**Review by beta-ziliani — COMMENTED** [review]
@straight-shoota how do you like this rewording?

**Pull request merged**